src/site/layouts/health_care_local_facility.drupal.liquid

```diff
@@ -187,6 +187,7 @@
                 {% include "src/site/facilities/facility_health_service.drupal.liquid" with
                     healthService = localHealthService
                     locationEntity = localService.entity
+                    mainFacilityPhoneNumber = fieldPhoneNumber
```

Comment:
I don't think we need this. We should already have the fieldPhoneNumber in the parent template. We might need to pass it down or `assign` it.

Reply:
isn't it exactly what I am doing `passing it` ?

Reply:
You added a new field to the GraphQL query that isn't needed.